sdk/documentintelligence/test-resources.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
Not required for the release but wanted to get the live tests pipeline working, if possible.